modules/openapi-generator/src/main/resources/csharp-netcore/api.mustache

```diff
@@ -256,36 +256,48 @@ namespace {{packageName}}.{{apiPackage}}
             if (localVarAccept != null) localVarRequestOptions.HeaderParameters.Add("Accept", localVarAccept);
 
             {{#pathParams}}
-            {{^vendorExtensions.x-csharp-value-type}}
+            {{#required}}
+            localVarRequestOptions.PathParameters.Add("{{baseName}}", {{packageName}}.Client.ClientUtils.ParameterToString({{paramName}})); // path parameter
+            {{/required}}
```

Comment:
It has been confirmed that required parameters have values at the beginning of the method regardless of whether they are nullable.